src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        final int maxNumber = 100;
```

Comment:
let's create a constant for 100 number
https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names